alacritty_terminal/src/ansi.rs

```diff
@@ -1007,6 +1011,25 @@ where
                 }
             },
 
+            // Hyperlink.
+            b"8" if params.len() > 2 => {
+                let link_params = params[1];
+                let uri = str::from_utf8(params[2]).unwrap_or_default();
+                if uri.is_empty() {
+                    self.handler.set_hyperlink(None);
+                    return;
+                }
```

Comment:
This should be made more clear, probably by adding a comment and newline as separation. It's not just some error handling, this is how hyperlinks are stopped.